eng/common-tests/matrix-generator/pr-matrix-samples/pr-matrix-generation-acceptance.helpers.ps1

```diff
@@ -0,0 +1,51 @@
+$PackagePropertiesPath = Join-Path "eng" "common" "scripts" "Save-Package-Properties.ps1"
+
+Function Get-Repo {
+    param (
+        [string]$Repo,
+        [string]$Reference
+    )
+
+    $tempDir = [System.IO.Path]::GetTempPath()
+    $repoPath = Join-Path $tempDir $Repo.Replace("/", "-")
+
+    if (!(Test-Path $repoPath)) {
+        New-Item -ItemType Directory -Path $repoPath | Out-Null
+
+        try {
+            Push-Location $repoPath
+            git clone --no-checkout "https://github.com/$Repo.git" .
```

Comment:
Probably worth logging at least since these can take a while depending on repo?

```suggestion
            Write-Host "git clone --no-checkout 'https://github.com/$Repo.git' ."
            git clone --no-checkout "https://github.com/$Repo.git" .
```

Reply:
Yeah I'll add that in the PR that fills this more.